### PR TITLE
feat/thumbs up down endpoints

### PR DESCRIPTION
**Pull request by ff6347**
This PR is the backend part for https://github.com/technologiestiftung/parla-frontend/pull/116

It adds a new route called feedback for POST and GET. Allows to update rows in user_requests with id of feedback in table feedbacks. Also updates the schema and sets RLS for table user_requests

-  feat(schema): Add tables for user feedback (7716180)
-  feat(feedback): Implement feedback GET and POST (ad47bbc)
-  fix: Remove duplicate types (78cdc66)
- fix: Error from merge conflict (feefa9e)
-  test: Remove unused test (4fc6893)
-  test(documentation): Remove test four doc route (cb9b645)
-  test(feedback): Add tests for feedback route (780ecfa)
-  test(feedback): Remove dynamic prop from snapshot (92eb068)
-  test(logging): Disable fastify logging for tests (610fa67)
-  chore: Update types (bacccd0)
-  chore: Make eslint happy (24b0b77)
-  chore(housekeeping): Remove unused code (ce1a451)
-  chore: Make eslint ignore generated file (3b42248)
-  Merge branch 'staging' into feat/thumbs-up-down-endpoints (b899c6a)
-  Merge branch 'staging' into feat/thumbs-up-down-endpoints (7a543d1)
-  docs(feedback): Hint about initial feedback insert (622f16a)

### CODE REVIEW

File: src/lib/routes/feedback-route.ts [hunk 1]
```diff
@@ -0,0 +1,134 @@
+import { FastifyInstance } from "fastify";
+import supabase from "../supabase.js";
+import { UserError } from "../errors.js";
+
+const feedbackRoute = function (
+	app: FastifyInstance,
+	options: unknown,
+	next: (err?: Error | undefined) => void,
+) {
+	// app.get("/", (req, res) => {
```
Comment (Jaszkowic) [inline src/lib/routes/feedback-route.ts:10]: Nitpick: Commented out code should be removed

**Review by Jaszkowic — DISMISSED** [review]
LGTM, comments should not block this PR

File: src/tests/app.test.ts [hunk 2]
```diff
@@ -22,17 +22,17 @@ test("default health check route should return 200", async (t) => {
...
-		method: "GET",
-		url: {
-			pathname: "/documentation/static/index.html",
-			hostname: "localhost",
-			port: 8888,
-			protocol: "http",
-		},
-	};
-	const response = await t.context.server.inject(opts);
+// test("documentation route should return 200", async (t) => {
```
Comment (Jaszkowic) [inline src/tests/app.test.ts:25]: Why was this unrelated test removed in this PR?

Comment (ff6347) [reply src/tests/app.test.ts:25]: The test did make all tests hang. Also I think the swagger UI is nice for development but should not land in production

File: src/lib/routes/feedback-route.ts (hunk 1 above)
Comment (ff6347) [reply src/lib/routes/feedback-route.ts:10]: yes

File: src/tests/app.test.ts (hunk 2 above)
Comment (Jaszkowic) [reply src/tests/app.test.ts:25]: I understand. Would remove the test completely

Comment (ff6347) [reply src/tests/app.test.ts:25]: Yes. Will do

**Pull request merged**